boards/sipeed-longan-nano/doc.txt

```diff
@@ -66,24 +66,37 @@ MCU pins and their configuration in RIOT.
 
 ## Flash the board
 
-The board is flashed via a JTAG interface with OpenOCD (at least [release version 0.12.0]
+### via DFU bootloader
```

Comment:
It may seem nitpicky, but I think headings in English should be upper case. Since `Via DFU Bootloader` seems a bit strange, we could use `Using DFU Bootloader` and `Using External Debug Adapter`.

Also the heading above should be changed, usually we use `Flashing the Device`. The heading used above is probably just a leftover.